config/config.go

```diff
@@ -46,183 +55,234 @@ const (
 	FlagSequencerRollupID = "rollkit.sequencer_rollup_id"
 	// FlagExecutorAddress is a flag for specifying the sequencer middleware address
 	FlagExecutorAddress = "rollkit.executor_address"
+
+	// FlagPrometheus is a flag for enabling Prometheus metrics
+	FlagPrometheus = "instrumentation.prometheus"
+	// FlagPrometheusListenAddr is a flag for specifying the Prometheus listen address
+	FlagPrometheusListenAddr = "instrumentation.prometheus_listen_addr"
+	// FlagMaxOpenConnections is a flag for specifying the maximum number of open connections
+	FlagMaxOpenConnections = "instrumentation.max_open_connections"
+
+	// FlagP2PListenAddress is a flag for specifying the P2P listen address
+	FlagP2PListenAddress = "p2p.listen_address"
+	// FlagP2PSeeds is a flag for specifying the P2P seeds
+	FlagP2PSeeds = "p2p.seeds"
+	// FlagP2PBlockedPeers is a flag for specifying the P2P blocked peers
+	FlagP2PBlockedPeers = "p2p.blocked_peers"
+	// FlagP2PAllowedPeers is a flag for specifying the P2P allowed peers
+	FlagP2PAllowedPeers = "p2p.allowed_peers"
+
+	// FlagEntrypoint is a flag for specifying the entrypoint
+	FlagEntrypoint = "entrypoint"
+	// FlagChainConfigDir is a flag for specifying the chain config directory
+	FlagChainConfigDir = "chain.config_dir"
 )
 
-// NodeConfig stores Rollkit node configuration.
-type NodeConfig struct {
+// RollkitConfig stores Rollkit node configuration.
+type RollkitConfig struct {
```

Comment:
```suggestion
type Config struct {
```
this is the rollkit repo, no need to call it rollkit